klp_tc_functions.sh

```diff
@@ -288,7 +288,7 @@ function klp_tc_cleanup() {
 }
 
 function klp_tc_init() {
-    trap "[ \$? -ne 0 ] && call_recovery_hooks; klp_tc_cleanup; echo TEST FAILED while executing \'\$BASH_COMMAND\'" EXIT
+    trap "ret=\$?; [ \$ret -ne 0 ] && call_recovery_hooks; klp_tc_cleanup; result=FAILED; [ \$ret -ne 99 ] && result=SKIPPED; echo TEST \$result while executing \'\$BASH_COMMAND\'" EXIT
```

Comment:
I may misread this, but shouldn't there be

[ \$ret -eq 99 ] && result=SKIPPED;

?

Reply:
Good catch. Also I had to add one more place where error code evaluation was needed.